axe_selenium_python/axe.py

```diff
@@ -93,17 +93,19 @@ def report(self, violations):
             string += "\n\n\n"
         return string
 
-    def write_results(self, data, name="results.json"):
+    def write_results(self, data, name):
         """
         Write JSON to file with the specified name.
 
         :param name: Name of file to be written to.
         :param output: JSON object.
         """
-        filepath = os.path.join(os.getcwd(), name)
+        filepath = os.path.abspath(name)
```

Comment:
If we set the default value of name to None, we could add a check to write to the current directory when name is undefined.

something like:
```
def write_results(self, data, name=None):
    filepath = os.path.abspath(name) if name else os.path.join(os.getcwd(), name)
```

Reply:
Done, I wrote the same over multiple lines because I got a "line to long" error in the linter when I wrote everything on one line.